### PR TITLE
Add "xdg-open" and "xsel -b" for X support

### LINKED ISSUE #2: add Linux support
The commands used to open the browser and to copy to clipboard are Mac-only.  It shouldn't be too hard to add fallbacks for Linux commands.  I'll see what I can whip up.

### PR DESCRIPTION
**Pull request by moopet**
This is a quick shim to make it work on systems running X instead of MacOS.

Closes #2

### CODE REVIEW

**Comment by jonathandion** [comment]
I was about to do it. Thanks @moopet

**Comment by mwcz** [comment]
Hah, I was working on this too. :P  Thanks!

**Comment by mwcz** [comment]
This will fix #2

**Comment by mwcz** [comment]
@moopet I'd recommend putting the "xdg-open" check first, like this:

```vim
if executable('xdg-open')                 
    call system('xdg-open ' . l:final_url)
elseif executable('open')                 
    call system('open ' . l:final_url)    
```
My system (Fedora 28) comes with /usr/bin/open which is the binary for openvt ("start a program on a new virtual terminal (VT)").

**Comment by moopet** [comment]
@mwcz yes, good idea, updated it.

File: plugin/githubinator.vim [hunk 1]
```diff
@@ -55,23 +55,27 @@ endfunction
 function! GithubOpenURL() range
     let l:final_url = s:generate_url()
 
-    if !executable('open')
-        echoerr 'githubinator: `open` command not found. Try `ghc` to copy URL'
-        return
+    if executable('xdg-open')
+        call system('xdg-open ' . l:final_url)
+      elseif executable('open')
```
Comment (danishprakash) [inline plugin/githubinator.vim]: @moopet it seems we're missing a few spaces here.

**Comment by prakashdanish** [comment]
Closing this, thanks for the PR @moopet

**Comment by danishprakash** [comment]
Merged, thanks for the PR @moopet

**Pull request merged**